integration/basic_test.go

```diff
@@ -380,9 +407,20 @@ func TestCollectionName(t *testing.T) {
 
 			assert.NoError(t, err)
 
+			// check collection name is in the list.
```

Comment:
```suggestion
			// check collection name is in the list
```
(sentence fragment - no dot at the end)

or

```suggestion
			// Check that collection name is in the list.
```
(full sentence with a dot at the end)

That's not important right now, but something to write down in the future.